common/models/transducer/alignment.py

```diff
@@ -64,3 +68,33 @@ def rnnt_alignment(source, **kwargs):
   _, alignment = tf_forward_shifted_rnnt(log_probs, targets.get_placeholder_as_batch_major(), enc_lens, dec_lens,
                                          blank_index=blank_idx, debug=False, with_alignment=True)
   return alignment  # (B, T)
+
+
+def rna_alignment_out_type(sources, **_kwargs) -> Data:
+  """ Computes the rna-alignment Data_out_type for RNA alignment
+  B: batch, T: time, U:target/labels, V: vocabulary
+  Args:
+      sources (list[LayerBase]): [output_log_prob, real_target, base:encoder]
+        output_log_prob.output (Data): (B, T, U+1, V) log-probabilities
+        real_target.output (Data): (B, U) -> [V] target labels
+        base:encoder.output (Data): (B, T, Feat) -> [V] encoder output
+  Returns:
+      (Data): Data for the alignment (B, T,)
+  """
+  return Data(name="rna_alignment_output", sparse=True, dim=sources[0].output.dim,
+              size_placeholder={0: sources[2].output.size_placeholder[0]})
+
+
+def rnnt_alignment_out_type(sources, **_kwargs) -> Data:
+  """ Computes the rnnt-alignment Data_out_type for RNNT alignment
+  B: batch, T: time, U:target/labels, V: vocabulary
+  Args:
+      sources (list[LayerBase]): [output_log_prob, real_target, base:encoder]
+        output_log_prob.output (Data): (B, T, U+1, V) log-probabilities
+        real_target.output (Data): (B, U) -> [V] target labels
+        base:encoder.output (Data): (B, T, Feat) -> [V] encoder output
+  Returns:
+      (Data): Data for the alignment (B, T+U,)
```

Comment:
Having the type above is maybe even better.
And then:

```suggestion
      sources:
        output_log_prob: (B, T, U+1, V) log-probabilities
        real_target: (B, U) -> [V] target labels
        base:encoder: (B, T, Feat) -> [V] encoder output
  Returns:
      alignment (B, T+U)
```